### PR TITLE
mingw : update to version 12.2.0-rt_v10-rev2

### PR DESCRIPTION
**Pull request by Baneeishaque**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Please use the `UCRT` variant. `MSVCRT` is the legacy one.

File: bucket/mingw.json [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-            "hash": "225e11db75cab7fb13054736332d078ce4f79f01bec2e1a2bffb13c1615d7f11",
+            "url": "https://github.com/niXman/mingw-builds-binaries/releases/download/12.2.0-rt_v10-rev2/i686-12.2.0-release-posix-dwarf-msvcrt-rt_v10-rev2.7z",
+            "hash": "a95b868d3fab77dbd96857c4c39fa5ccf516ebc00e6ea900a91f529d529e2fd0",
             "extract_dir": "mingw32"
         }
     },
     "post_install": "Copy-Item \"$dir\\bin\\mingw32-make.exe\" \"$dir\\bin\\make.exe\"",
     "env_add_path": "bin",
     "checkver": {
         "url": "https://github.com/niXman/mingw-builds-binaries/releases",
```
Comment (niheaven) [inline bucket/mingw.json:21]:
```suggestion
        "github": "https://github.com/niXman/mingw-builds-binaries",
```

File: bucket/mingw.json [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
-        "regex": "Release ([\\d.]+)-(?<build>[a-z0-9_\\-]+)",
+        "regex": "Release of ([\\d.]+)-(?<build>[a-z0-9_\\-]+)",
```
Comment (niheaven) [inline bucket/mingw.json:22]:
```suggestion
        "regex": "tag/([\\d.]+)-(?<build>[a-z0-9_\\-]+)",
```

**Review by niheaven — CHANGES_REQUESTED** [review]

**Pull request merged**